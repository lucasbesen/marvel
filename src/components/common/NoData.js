import * as React from 'react';
import styled from 'styled-components';

import { Content } from '.';

const Text = styled.div`
  display: flex;
  flex: 1;
  height: 100%;
  justify-content: center;
  align-items: center;
  align-self: center;
`;

const NoData = (): React.ReactNode => (
  <Content>
    <Text>No Data</Text>
  </Content>
);

export default NoData;
