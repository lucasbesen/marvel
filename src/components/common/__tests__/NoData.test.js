import React from 'react';
import renderer from 'react-test-renderer';

import NoData from '../NoData';

it('should render NoData without error', () => {
  const component = renderer.create(<NoData />);

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
