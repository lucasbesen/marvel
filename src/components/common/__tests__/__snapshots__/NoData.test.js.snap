// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NoData without error 1`] = `
<div
  className="MuiPaper-root WithStyles(ForwardRef(Paper))-root-1 MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="sc-bdVaJa hNtgBK"
  >
    No Data
  </div>
</div>
`;
